### PR TITLE
docs(teams): revamp to include organizations

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Description

- docs(teams): revamp to include organizations

### CODE REVIEW

File: src/content/docs/teams.mdx [hunk 1]
```diff
@@ -1,46 +1,138 @@
...
-description: Working with a team of developers
+description: Learn how to manage teams and organizations in Shorebird
 ---
 
-Shorebird Teams tier and higher allows adding an unlimited number of
-collaborators to an app. Shorebird free "Hobby" users can upgrade
-to the Teams tier to add collaborators via a link in the
-[Shorebird console](https://console.shorebird.dev).
+Shorebird's Pro tier and unlocks new features for collaborating with your team
+include:
```
Comment (bryanoltman) [inline src/content/docs/teams.mdx:7]:
```suggestion
Shorebird's Pro tier unlocks new features for collaborating with your team that
include:
```

File: src/content/docs/teams.mdx [hunk 2]
```diff
@@ -1,46 +1,138 @@
...
+### Organization Owner
+
+There is only one owner per organization (the person who created the
+organization) and they have full control over the organization and all apps
+within it.
+
+The owner can:
+
+- Everything an Admin can do
+- Manage the billing information for the organization
```
Comment (eseidel) [inline src/content/docs/teams.mdx]: There is an upsell opportunity to link to the pricing page and/or mention "If you need more control than this, consider our enterprise plan" or similar.

File: src/content/docs/teams.mdx [hunk 3]
```diff
@@ -65,79 +157,29 @@ The Admin role can:
 - Everything a Developer can do
 - Add and Remove Developers for an App
 - Delete Apps
-- Delete Releases
-  Admin permissions cannot currently be modified in the console. Please email us
-  if you need to add/remove Admins from an app.
+- Delete Releases Admin permissions cannot currently be modified in the console.
```
Comment (eseidel) [inline src/content/docs/teams.mdx]: typo?

File: src/content/docs/teams.mdx [hunk 4]
```diff
@@ -65,79 +157,11 @@ The Admin role can:
...
-Owner changed) by contacting us at contact@shorebird.dev.
-
-## Sharing Multiple Apps
-
-Each app maintains a separate list of collaborators. If you need to share
-multiple apps with a team, you will need to add each app separately. We plan
-to add more robust teams support later this year.
-https://github.com/shorebirdtech/shorebird/issues/739
+- Delete Releases Admin permissions cannot currently be modified in the console.
+  Please email us if you need to add/remove Admins from an app.
```
Comment (bryanoltman) [inline src/content/docs/teams.mdx]: Typo here

File: astro.config.mjs [hunk 5]
```diff
@@ -125,8 +125,8 @@ export default defineConfig({
           link: '/flutter-version',
         },
         {
-          label: 'Teams',
-          link: '/teams',
+          label: 'Organizations',
+          link: '/organization',
```
Comment (eseidel) [inline astro.config.mjs:129]: /orgs? maybe

File: src/content/docs/orgs.mdx [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+---
+
+Shorebird's Pro tier unlocks new features for collaborating with your peers
+including:
+
+- Create unlimited organizations
+- Add unlimited members to your organizations
+- Manage roles for your organization members
+- Add unlimited collaborators to an individual app
+- Manage roles for your app collaborators
```
Comment (bryanoltman) [inline src/content/docs/orgs.mdx:13]: I'm not sure what the proper grammatical term is for this, but I feel like "collaborating" and "create" should have the same verb form (either collaborate+create or collaborating+creating)

File: src/content/docs/orgs.mdx [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+There is only one owner per organization (the person who created the
+organization) and they have full control over the organization and all apps
+within it.
+
+The owner can:
+
+- Do everything an Admin can do
+- Manage the billing information for the organization
+
+If you need more control, consider our [enterprise plan](https://shorebird.dev/pricing).
```
Comment (eseidel) [inline src/content/docs/orgs.mdx:119]:
```suggestion
If you need more control or custom roles, consider our [enterprise plan](https://shorebird.dev/pricing).
```

File: src/content/docs/orgs.mdx (hunk 6 above)
Comment (bryanoltman) [inline src/content/docs/orgs.mdx:13]:
```suggestion
- Creating unlimited organizations
- Adding unlimited members to your organizations
- Managing roles for your organization members
- Adding unlimited collaborators to an individual app
- Managing roles for your app collaborators
```

Comment (bryanoltman) [reply src/content/docs/orgs.mdx]: Or:

```
Shorebird's Pro tier unlocks new features for collaborating with your peers. With the Shorebird Pro tier, you can:
```

**Review by bryanoltman — APPROVED** [review]

File: src/content/docs/orgs.mdx [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+
+Existing collaborators can be removed by clicking the trash icon next to their
+email address.
+
+## Collaborator Roles
+
+_Roles are per-app._ Each account can create Apps tied to that account. Once an
+App is created, access to that app can be shared. If you have any questions
+about access control for your app, please do not hesitate to reach out to
+contact@shorebird.dev.
```
Comment (eseidel) [inline src/content/docs/orgs.mdx:142]: This is old.

Collaborators are used for per-app access controls.  Typically used to provide someone temporary access to only one of your applications.

Collaborators allow you to provide access to one of your apps without granting someone access to all of the apps in an organization.

Collaborators can be added both to apps within your personal storage as well as on any organization which you are an admin member of.

Collaborators can see the existence of an organization and its name, but cannot see any other aspects of an organization.

**Pull request merged**